docs/community_resources.md

```diff
@@ -1,6 +1,12 @@
 # Community Resources
 
-This is a currated list of various bots and helpfull tools that aim to making approaching Conan-Center-Index and contributing easier.
+This is a curated list of various bots and helpful tools that aim to making approaching Conan-Center-Index and contributing easier.
+
+<!-- toc -->
```

Comment:
Does this do anything? It does not look any different in GitHub MD render

Reply:
There are two 2 typos fixed in that line: currated and helpfull

Reply:
Sorry the GitHub UI shows the previous lines .... My question are only about the HTML comment.

```
<!-- toc -->
```

Reply:
[sorry, this got stuck as "pending" for 6 hours...]

Ah right, now I understand...

The table of contents is inserted between a pair of HTML comments: this `<!-- toc -->` and the `<!-- endToc -->`

Next time mdsnippets runs, it uses that pair of comments to work out where to place the updated ToC, if the headings have changed.

Is that clearer now?

Reply:
I am not sure CCI teams wants too many third-party extensions

Reply:
> I am not sure CCI teams wants too many third-party extensions

In https://github.com/conan-io/conan-center-index/issues/4367 I tried to explain how this would work in advance, to give people a chance to say if this approach was not appropriate, in advance of spending time on it...

I thought the comments were positive.

Reply:
Hmm theres no feed specifically on the tool... regardless I think it's fine for the toc thanks for the link